deploy/kubernetes/nebula_exporter.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: nebula-stats-exporter
+rules:
+  - apiGroups: [""]
+    resources:
+      - pods
+    verbs: ["get", "list", "watch"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: nebula-stats-exporter
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: nebula-stats-exporter
+subjects:
+  - kind: ServiceAccount
+    name: nebula-stats-exporter
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nebula-stats-exporter
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nebula-stats-exporter
+  namespace: default
+  labels:
+    k8s-app: nebula-stats-exporter
+spec:
+  selector:
+    matchLabels:
+      k8s-app: nebula-stats-exporter
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        prometheus.io/port: "9100"
+        prometheus.io/scrape: "true"
+      labels:
+        k8s-app: nebula-stats-exporter
+    spec:
+      serviceAccountName: nebula-stats-exporter
+      containers:
+        - name: exporter
+          image: vesoft/nebula-stats-exporter:v0.0.1
+          imagePullPolicy: Always
+          args:
+            - "-listen-address=0.0.0.0:9100"
+            - "-namespace=default"
```

Comment:
Is the namespace `default` not `nebula` ?

Reply:
for k8s, default namespace is 'default' is natural

Reply:
Shall we need to create a k8s namespace like `nebula` to separate from other services?

Reply:
no need, helm chart or operator will specify this namespace